### PR TITLE
734 jordans updates

### PR DESCRIPTION
**Pull request by nikeshashar**
Starting work on small UX updates with feedback from Mark and Jordan.

### CODE REVIEW

**Comment by sjmog** [comment]
One comment @nikeshashar

File: source/sass/_typography.scss [hunk 1]
```diff
@@ -205,7 +205,6 @@ picture {
   text-decoration: none;
   position: relative;
   &:after {
```
Comment (nikeshashar) [inline source/sass/_typography.scss]: @sjmog sure

File: source/payment.html.haml [hunk 2]
```diff
@@ -44,19 +60,7 @@
...
-        %h4 Bank Transfer
-      .column
-        = image_tag "payment-options/transferwise.jpg", class: "image--medium"
-        %h4 TransferWise
-      .column
-        = image_tag "payment-options/gocardless.jpg", class: "image--medium"
-        %h4 GoCardless
+
+= partial :"benefits_table"
+
```
Comment (jordanpoulton) [inline source/payment.html.haml]: Remove comparison table from this page.

**Comment by jafrog** [comment]
:shipit:

**Pull request merged**